ctapipe/utils/datasets.py

```diff
@@ -38,7 +38,7 @@ def get_searchpath_dirs(searchpath=os.getenv("CTAPIPE_SVC_PATH")):
     else:
         searchpaths = [Path(p) for p in os.path.expandvars(searchpath).split(":")]
 
-    searchpaths.append(get_cache_path(""))
+    searchpaths.append(get_cache_path(DEFAULT_URL))
```

Comment:
This doesn't seem like it should be part of this PR (and same for all the other changes to datasets.py)